src/crystal/system/event_loop.cr

```diff
@@ -4,7 +4,19 @@ abstract class Crystal::EventLoop
     {% if flag?(:wasi) %}
       Crystal::Wasi::EventLoop.new
     {% elsif flag?(:unix) %}
-      Crystal::LibEvent::EventLoop.new
+      # TODO: dragonfly: the kqueue evloop doesn't work properly (regular hangs on evloop.run)
+      # TODO: netbsd: the kqueue evloop doesn't work (needs investigation)
+      # TODO: openbsd: the kqueue evloop is sluggish when running std specs (the main fiber keeps reenqueueing itself from the evloop run)
+      # TODO: solaris: the epoll evloop hasn't been tested (better alternative: use event-ports)
+      {% if flag?("evloop=libevent") %}
```

Comment:
**polish:** The TODOs don't make much sense at this place anymore because there's no direct reference to these flags.
I'd suggest to move them either into the `epoll`/`kqueue` branches or into the else branch with a comment that this is a catch-all for all platforms where the system selector has not yet been enabled.

Reply:
@ysbaddaden The latest commit removes the todos entirely. I don't think they should be lost?

Reply:
Yes, I'll open issues for each target instead, following the bugs/regressions part of the PR description, and where we can lead investigations.